api/app/signals/apps/api/serializers/signal.py

```diff
@@ -514,6 +514,11 @@ def create(self, validated_data):
 
         if attachments:
             Signal.actions.copy_attachments(data=attachments, signal=signal, created_by=logged_in_user.email)
+            # Add history entries for every attachment that was copied. Only photos allowed for now.
+            for attachment in Attachment.objects.filter(_signal=signal).order_by('created_at'):
+                filename = os.path.basename(attachment.file.name)
+                msg = f'Foto gekopieerd van hoofdmelding: {filename}'
```

Comment:
Maybe also add who triggered this? So that we can see who was responsible for copying the attachments?

Reply:
Thanks for doing the check!